build.gradle.kts

```diff
@@ -1,38 +1,80 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id("org.springframework.boot") version "2.1.5.RELEASE"
-    id("io.spring.dependency-management") version "1.0.7.RELEASE"
-    kotlin("jvm") version "1.3.50"
-    kotlin("plugin.spring") version "1.3.50"
+    id("org.springframework.boot") version "2.2.4.RELEASE"
+    id("io.spring.dependency-management") version "1.0.9.RELEASE"
+    kotlin("jvm") version "1.3.61"
+    kotlin("plugin.spring") version "1.3.61"
 }
 
-group = "ch.hevs.cloudio2"
-version = "0.0.1-SNAPSHOT"
+group = "ch.hevs.cloudio"
+version = "0.2.0-SNAPSHOT"
 java.sourceCompatibility = JavaVersion.VERSION_1_8
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
+    implementation("org.springframework.boot:spring-boot-starter")
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.springframework.boot:spring-boot-starter-amqp")
     implementation("org.springframework.boot:spring-boot-starter-data-mongodb")
-    implementation("org.influxdb:influxdb-java")
-    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.9.9")
     implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("org.influxdb:influxdb-java")
+    implementation("org.bouncycastle:bcpkix-jdk15on:1.64")
+
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
-    implementation("org.bouncycastle:bcpkix-jdk15on:1.57")
+
     testImplementation("org.springframework.boot:spring-boot-starter-test")
-    testImplementation("org.jetbrains.kotlin:kotlin-test:1.3.50")
+    testImplementation("org.jetbrains.kotlin:kotlin-test:1.3.61")
     testImplementation("org.junit.jupiter:junit-jupiter-api")
 }
 
 tasks.withType<KotlinCompile> {
     kotlinOptions {
         freeCompilerArgs = listOf("-Xjsr305=strict")
-        jvmTarget = "1.8"
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+}
+
+tasks.register("bootRunDev") {
+    group = "application"
+    description = "Runs this project as a Spring Boot application with the cloudio dev environment"
+    dependsOn("cloudio-dev-environment:createDevServices")
+    doFirst {
+        tasks.bootRun.configure {
+            environment("cloudio.caCertificate", file("cloudio-dev-environment/certificates/ca.cer").readText())
+            environment("cloudio.caPrivateKey", file("cloudio-dev-environment/certificates/ca.key").readText())
+            environment("cloudio.caCertificateJksPath", "cloudio-dev-environment/certificates/ca.jks")
+            environment("cloudio.caCertificateJksPassword", "123456")
+
```

Comment:
Need those two parameters for influx batch

cloudio.influxBatchIntervallMs=1000
cloudio.influxBatchSize=1

Reply:
also need server.port=8081 for the RESTful api